ROFO-42 피드백 반영

Unagi-zoso opened the pull request:
### 개요
피드백 주신 부분 반영했습니다.

### 변경사항
- Validator 테스트 추가했습니다.
- Kakao API 인터페이스 이름 명시적으로 변경
- 공통된 로직 함수로 하나로 모음


### 테스트
- 테스트 코드 추가여부 O
O

### 관련 지라 및 위키 링크
 - [JIRA](https://weit-2nd.atlassian.net/browse/ROFO-42) 


### 리뷰어에게 하고 싶은 말
잘 부탁드리겠습니다.

Unagi-zoso committed:
ROFO-42 사용되지 않는 클래스 제거

Unagi-zoso committed:
ROFO-42 MultipartFile 생성 편의 함수 작성 및 중복적인 부분 제거

Unagi-zoso committed:
ROFO-42 코드 스타일 개선 및 로깅 제거

Unagi-zoso committed:
ROFO-42 Kakao API 인터페이스 이름 명시적으로 변경

Unagi-zoso committed:
ROFO-42 AuthController 문서 작성 및 입력 사진 1MB 이상 시 테스트 추가

Unagi-zoso committed:
ROFO-42 Validator 테스트 추가

Unagi-zoso committed:
Merge branch 'develop' into feature/ROFO-42

Unagi-zoso committed:
ROFO-42 회원가입 API 문서 최신화

konempty commented on src/test/kotlin/kr/weit/roadyfoody/auth/fixture/AuthFixture.kt:
```diff
@@ -1,47 +1,18 @@
 
 const val TEST_IMAGE_FILE_NAME = "testImage"
 
 fun createTestImageFile(
```
얘도 MultipartFileHelper.kt로 보내주는게 어떨까요??

konempty reviewed (approved):
고생하셨습니다 👍 👍 
리뷰 반영을 별도의 작업으로 때서 하셨군요 ㅋㅋㅋㅋ

Unagi-zoso committed:
ROFO-42 createTestImageFile 메소드 위치 이동 -> MultipartFileHelper

테스트 시 여러 기능에서 이미지를 만들 경우가 많아보여
좀 더 공용으로 사용할 수 있는 위치에 뒀습니다.

근데 MultipartFileHelper 라는 이름이 적합한지는 조금 의문이 듭니다.
공용 Fixture 의 이름을 어떻게 지을까 고민하다 정하긴 했지만 더 좋은 이름이 있다면 바꾸는 것도 괜찮아보입니다.

lin-chae commented on src/test/kotlin/kr/weit/roadyfoody/auth/presentation/api/AuthControllerTest.kt:
```diff
@@ -91,13 +110,41 @@ class AuthControllerTest(
             }
 
+            `when`("파일의 크기가 1MB를 초과하면") {
+                every { authCommandService.register(any<String>(), any<SignUpRequest>(), any<MultipartFile>()) } just runs
```
저도 이 테스트 코드를 짜고 있었는데 잘 짜주셨네요! 참고하겠습니다 :)

lin-chae reviewed (approved):
많은 변경이 있네요! 고생하셨습니다 👍
